webpack.config.js

```diff
@@ -25,6 +25,7 @@ module.exports = function(env, argv) {
         "util": require.resolve("util/"),
         "url": require.resolve("url/"),
         "vm": require.resolve("vm-browserify"),
+        "querystring": require.resolve('querystring-es3'),
```

Comment:
Please add a comment for why this change is needed.

Reply:
Yeah @sreenara 
Webpack 5 no longer includes Node.js polyfills by default, which is causing the error for the querystring module. we need to add a fallback configuration for querystring in webpack.config.js file.